### PR TITLE
Tweak lexicon descriptions

### PR DESCRIPTION
**Pull request by emilyliu7321**
We want to auto-generate some specs off of these lexicons (e.g. OpenAPI), so this PR is to standardize lexicon descriptions to make auto-generation easier down the line.

### CODE REVIEW

File: lexicons/app/bsky/actor/getProfiles.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "defs": {
     "main": {
       "type": "query",
+      "description": "Get detailed profile views of multiple actors",
```
Comment (devinivy) [inline lexicons/app/bsky/actor/getProfiles.json:7]:
```suggestion
      "description": "Get detailed profile views of multiple actors.",
```

**Review by devinivy — APPROVED** [review]
This rules!  Would love to standardize a little style guide for writing these.

File: lexicons/app/bsky/actor/getProfiles.json (hunk 1 above)
Comment (emilyliu7321) [reply lexicons/app/bsky/actor/getProfiles.json:7]: ooh good catch 🙏

**Comment by emilyliu7321** [comment]
@devinivy done! 🙏

**Pull request merged**